tests/suite/manifests/snippets-filter/valid-sf.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: gateway.nginx.org/v1alpha1
```

Comment:
Any reason not to include the coffee route and grpc route in this file instead of having them in their own files? It would then be consistent with all the others.

Reply:
Not necessarily. I can update them to be consistent.